src/library_browser.js

```diff
@@ -225,6 +225,33 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if WASM && MAIN_MODULE
```

Comment:
our preprocessor is pretty simple, and doesn't support `&&`... can nest one if inside another.

Reply:
(I think that explains the CI errors)